### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by Dorcas126**
In this project, I use a relational database to create the initial data structure for a vet clinic. Icreate a table to store animals' information, insert some data into it, and query it.

#Project description

- Use [this template](https://github.com/microverseinc/curriculum-template-databases) to generate repo.
- Create a database named vet_clinic.
- Create a table named animals with the following columns:
  - id: integer
  - name: string
  - date_of_birth: date
  - escape_attempts: integer
  - neutered: boolean
  - weight_kg: decimal
- Copy the SQL I  used in the previous point into the schema.sql file in the generated repository.
- Insert the following data:
   - Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
  - Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
  - Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
  - Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
- Copy the SQL I used in the previous point into the data.sql file in the generated repository.
- Write queries for the following quests:
   - Find all animals whose name ends in "mon".
  - List the name of all animals born between 2016 and 2019.
  - List the name of all animals that are neutered and have less than 3 escape attempts.
  - List the date of birth of all animals named either "Agumon" or "Pikachu".
  - List name and escape attempts of animals that weigh more than 10.5kg
  - Find all animals that are neutered.
  - Find all animals not named Gabumon.
  - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)
- Take a screenshot of the results of queries.
- Copy the SQL I used in the previous point into the queries.sql file in the generated repository.

image

image

image

image

image

image

image

image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES (1,'Agumon','3-02-2020', 0, true,10.3);
```
Comment (mckent05) [inline data.sql]: - Good job done so far, but according to the project requirements, the weight for `Agumon` should be `10.23kg`, please correct this 🙏

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,58 @@
...
+animals 
+WHERE 
+name LIKE '%mon';
+
+SELECT
+name
+FROM 
+animals
+ WHERE
+ date_of_birth BETWEEN 2016 AND 2019;
```
Comment (mckent05) [inline queries.sql]: - Good job done so far, but according to the project requirements, you are to attach the screenshots for all queries, I cannot find the screenshot for this query. Please fix this 🙏

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,58 @@
...
+ animals
+ WHERE
+ neutered = true;
+
+ SELECT
+ *
+ FROM
+ animals
+ WHERE
+ name <> Gabumon;
```
Comment (mckent05) [inline queries.sql]: - Good job done so far, but according to the project requirements, you are to attach the screenshots for all queries, I cannot find the screenshot for this query. Please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Dorcas126, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job creating your animals table 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @Dorcas126, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**